lib/ansible/cli/__init__.py

```diff
@@ -730,11 +727,10 @@ def cli_executor(cls, args=None):
             ansible_dir = Path(C.ANSIBLE_HOME).expanduser()
             try:
                 ansible_dir.mkdir(mode=0o700)
+            except FileExistsError:
+                display.debug(f"The {ansible_dir!r} directory already exists")
```

Comment:
Use `exist_ok=True` instead of catching the exception:

```suggestion
                ansible_dir.mkdir(mode=0o700, exist_ok=True)
```